src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,29 @@
-export const Tabs = () => {};
+import cn from 'classnames';
+
+export const Tabs = ({ goods, currentTab, set }) => {
```

Comment:
The `Tabs` component is currently using props `goods`, `currentTab`, and `set`. According to the task requirements, it should use `tabs`, `activeTabId`, and `onTabSelected`. Please update these prop names to match the expected ones.